src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -304,6 +269,72 @@ g.test('sampleCount,valid_sampleCount_with_other_parameter_varies')
     }, !success);
   });
 
+g.test('texture_size,default_value_and_smallest_size,uncompressed_format')
```

Comment:
I haven't reviewed these two new tests just yet. Would you mind putting them in a separate PR? I think they should separate cleanly.

Reply:
Done.